docs/02-GettingStarted.md

```diff
@@ -5,6 +5,30 @@ and bootstrapped your first test suites. Codeception has generated three of them
 They are well described in the [previous chapter](http://codeception.com/docs/01-Introduction). Inside your __/tests__ folder you will have three `.yml` config files and three directories
 with names corresponding to these suites: `unit`, `functional`, `acceptance`. Suites are independent groups of tests with a common purpose.
 
+## The Codeception Syntax
+
+Codeception follows simple naming rules to make it easy to remember (as well as easy to understand) its method names.
+
+* **Actions** start with a plain english verb, like "click" or "fill". Examples:
+    ```php
```

Comment:
Done. But I don't like it, since this way it looks like you can use the shown code right below the opening `<?php` tag....